app/src/main/java/de/westnordost/streetcomplete/overlays/mtb_scale/MtbScaleOverlayForm.kt

```diff
@@ -0,0 +1,109 @@
+package de.westnordost.streetcomplete.overlays.mtb_scale
+
+import android.content.Context
+import android.os.Bundle
+import android.view.View
+import androidx.core.view.isGone
+import com.russhwolf.settings.ObservableSettings
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.UpdateElementTagsAction
+import de.westnordost.streetcomplete.databinding.FragmentOverlayMtbScaleSelectBinding
+import de.westnordost.streetcomplete.overlays.AbstractOverlayForm
+import de.westnordost.streetcomplete.util.LastPickedValuesStore
+import de.westnordost.streetcomplete.util.ktx.valueOfOrNull
+import de.westnordost.streetcomplete.util.logs.Log
+import de.westnordost.streetcomplete.view.setImage
+import org.koin.android.ext.android.inject
+
+class MtbScaleOverlayForm : AbstractOverlayForm() {
```

Comment:
A lot of code here and in that ViewController. Why can this not inherit from `AImageSelectOverlayForm` like the `WayLitForm`?